dashboard/app/models/standalone_video.rb

```diff
@@ -0,0 +1,51 @@
+# == Schema Information
+#
+# Table name: levels
+#
+#  id                       :integer          not null, primary key
+#  game_id                  :integer
+#  name                     :string(255)      not null
+#  created_at               :datetime
+#  updated_at               :datetime
+#  level_num                :string(255)
+#  ideal_level_source_id    :integer
+#  solution_level_source_id :integer
+#  user_id                  :integer
+#  properties               :text(65535)
+#  type                     :string(255)
+#  md5                      :string(255)
+#  published                :boolean          default(FALSE), not null
+#  notes                    :text(65535)
+#
+# Indexes
+#
+#  index_levels_on_game_id  (game_id)
+#
+
+class StandaloneVideo < Level
+  serialized_attrs %w(
+    skip_dialog
+    skip_sound
+  )
+
+  before_validation do
+    self.skip_dialog = true
+    self.skip_sound = true
+  end
```

Comment:
They are serialized attributes — they're always true for `StandaloneVideo` levels but not _all_ levels of any type.